validator.js

```diff
@@ -49,6 +49,10 @@ function _arrayWithHoles(arr) {
 }
 
 function _iterableToArrayLimit(arr, i) {
+  if (!(Symbol.iterator in Object(arr) || Object.prototype.toString.call(arr) === "[object Arguments]")) {
```

Comment:
how did this change come about? 🤔

Reply:
https://github.com/validatorjs/validator.js/pull/1107#issuecomment-532114148